.github/release.yml

```diff
@@ -0,0 +1,20 @@
+changelog:
+  categories:
+    - title: New features
+      labels:
```

Comment:
Why do we assume here that there can be more labels for one category? Is it really necessary? 
Then you need to strip `- ` from labels in the shell script.